index.js

```diff
@@ -1,8 +1,59 @@
+const assert = require('assert')
 const semverSatisfies = require('semver/functions/satisfies')
 const { convertToCsvString } = require('./src/csv')
 const bitbucket = require('./src/bitbucket')
 const github = require('./src/github')
 
+/**
+ * Get the security vulnerabilities for the specified version control systems.
+ *
+ * If a Bitbucket workspace is specified, it will be scanner.
+ *
+ * If a GitHub organization is specified, it will be scanner.
+ *
+ * @param {{
+    bitbucket: { username: string?, appPassword: string?, workspace: string? },
+    github: { token: string, organization: string? }
+    }} config
+ * @returns {Promise<[]>}
+ */
+const getSecurityVulnerabilitiesForMultiple = async config => {
+  const bbConfig = config.bitbucket || {}
+  const ghConfig = config.github || {}
+  
+  assert(ghConfig.token, 'A GitHub token is required to check for vulnerabilities')
+  
+  assert(
+    bbConfig.workspace || ghConfig.organization,
+    'You must specify a GitHub organization, a Bitbucket workspace, or both'
+  )
+  
+  const allVulnerabilities = []
+  
+  if (bbConfig.workspace) {
```

Comment:
whats the type on this prop, obj or prim?

Reply:
If they legitimately provided one, it will be a non-empty string. If not, it will either be undefined or an empty string, I believe.

Reply:
👍

Reply:
So to answer the question you actually asked, it's a primitive

Reply:
yep, perfect then